index.html

```diff
@@ -0,0 +1,521 @@
+        </p>
+        <p class="content">
+            Inflation has been the workhorse of cosmology for 35 years, though many, including Dr. Guth, wondered whether it could ever be proved.
+        </p>
+        <p class="content">
+            If corroborated, Dr. Kovac’s work will stand as a landmark in science comparable to the recent discovery of dark energy pushing the universe apart, or of the Big Bang itself. It would open vast realms of time and space and energy to science and speculation.
+        </p>
+        <p class="content">
+            Confirming inflation would mean that the universe we see, extending 14 billion light-years in space with its hundreds of billions of galaxies, is only an infinitesimal patch in a larger cosmos whose extent, architecture and fate are unknowable. Moreover, beyond our own universe there might be an endless number of other universes bubbling into frothy eternity, like a pot of pasta water boiling over.
+        </p>
+        <h3 class="content bolder">‘As Big as It Gets’</h3>
+        <p class="content">
+            In our own universe, it would serve as a window into the forces operating at energies forever beyond the reach of particle accelerators on Earth and yield new insights into gravity itself. Dr. Kovac’s ripples would be the first direct observation of gravitational waves, which, according to Einstein’s theory of general relativity, should ruffle space-time.
+        </p>
+        <p class="content">
+            Marc Kamionkowski of Johns Hopkins University, an early-universe expert who was not part of the team, said, “This is huge, as big as it gets.”
+        </p>
+        <p class="content">
+            He continued, “This is a signal from the very earliest universe, sending a telegram encoded in gravitational waves.”
+        </p>
+        <p class="content">
+            The ripples manifested themselves as faint spiral patterns in a bath of microwave radiation that permeates space and preserves a picture of the universe when it was 380,000 years old and as hot as the surface of the sun.
+        </p>
+        <p class="content">
+            Dr. Kovac and his collaborators, working in an experiment known as Bicep, for Background Imaging of Cosmic Extragalactic Polarization, reported their results in a scientific briefing at the Center for Astrophysics here on Monday and in a set of papers submitted to The Astrophysical Journal.
+        </p>
+        <br><br>
+        <!-- inflation section here -->
+        <div class="inflation-container">
+            <h4>The Theory of Inflation</h4>
```

Comment:
- [ ] Make sure 'The Theory of Inflation' section is centered horizontally. It is currently leaning to the left edge of the browser.